clang/test/Sema/attr-ms-noinline.cpp

```diff
@@ -0,0 +1,72 @@
+// RUN: %clang_cc1 -verify -fsyntax-only %s -Wno-c++17-extensions
```

Comment:
This test is a copy/paste of the other one, I'd rather see this added into the other test/merged instead to show off both spellings in one place (particularly because that is attr-noinline.cpp).

Reply:
Okay, will fix.

Reply:
`[[msvc::noinline]]` is just an alias of `[[clang::noinline]]`, and maybe we don't really need UT for it, so I just deleted the redundant test files. What do you think?

Reply:
I removed the redundant unit tests, but no new test cases were added. I'm not sure if that's acceptable, because `[[msvc::noinline]]` is just an alias for `[[clang::noinline]]`. What do you think?